apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.tsx

```diff
@@ -169,6 +171,11 @@ function WelcomeTour() {
 		},
 	};
 
+	// Theme isn't immediately available, so we prevent rendering so the content doesn't switch after it is presented, since some content is based on theme
+	if ( null === themeName ) {
```

Comment:
Does this still show automatically after you onboard with a new site? Not sure what you mean by `immediately` :D

Reply:
`immediately` just means on first render of the component. I was seeing the original coffee cup welcome image and then a second or 2 later the site info / theme request came back and populated the `themeName` variable and then the videomaker image popped in.